process/process.go

```diff
@@ -196,21 +185,56 @@ func (p *Process) HandleMessage(m Message) {
 		p.handlePrevote(m)
 	case *Precommit:
 		p.handlePrecommit(m)
+	case *Resync:
+		p.handleResync(m)
 	}
 }
 
-func (p *Process) resend(height block.Height, round block.Round) {
+func (p *Process) resend(to id.Signatory, height block.Height, round block.Round) {
```

Comment:
It is more idiomatic to have `to` be a pointer, and then to check against `nil` instead of against the `0x0000..` signatory. It is also slightly faster (albeit, negligibly). It doesn't actually bother me that much though, however, either way we should comment this function and document the specifics of `to` and how it alters the behaviour of the function / what are its valid values.